test/ETO/ETOCommitment.js

```diff
@@ -416,18 +428,36 @@ contract("ETOCommitment", ([, admin, company, nominee, ...investors]) => {
 
     it("with changing ether price during ETO", async () => {
       await skipTimeTo(publicStartDate.add(1));
+      // set some round ETH price to minimize rounding errors in the contract
+      await gasExchange.setExchangeRate(etherToken.address, euroToken.address, Q18, {
+        from: admin,
+      });
       const tokenprice = tokenTermsDict.TOKEN_PRICE_EUR_ULPS;
-      const ticket = Q18.mul(107.61721).add(1);
+      const ticket = Q18.mul("1070.61721").add(1);
```

Comment:
There is an extra zero here (which doesnt really matter), but again, there are a lot of other places below where we multiply with a decimal-number and it is not using quotes

e.g. l. 1648, 1848, 2810 and prob more